scipy/spatial/distance.py

```diff
@@ -1322,6 +1363,45 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
 
          dm = pdist(X, 'sokalsneath')
 
```

Comment:
IMHO, placing the calling conventions in the notes is cleaner. I would stick with the current code.